### PR TITLE
Cleanup ActiveIssue style

### PR DESCRIPTION
**Pull request by sdmaclea**
This is using a regular expression search and replace to switch the ActiveIssue style on existing disabled tests.  Please check that I did this correctly.

### CODE REVIEW

**Review by safern — APPROVED** [review]
Skimmed through. Looks good as long as CI is green. Thanks for the cleanup!

File: src/libraries/System.Net.HttpListener/tests/HttpResponseStreamTests.cs [hunk 1]
```diff
@@ -351,7 +353,8 @@ public async Task Write_TooLittleAsynchronouslyAndClose_ThrowsInvalidOperationEx
             }
         }
 
-        [ConditionalFact(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))] // [ActiveIssue("https://github.com/dotnet/runtime/issues/21918", TestPlatforms.AnyUnix)]
+        [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/21918", TestPlatforms.AnyUnix, nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))]
```
Comment (ViktorHofer) [inline src/libraries/System.Net.HttpListener/tests/HttpResponseStreamTests.cs:357]:
```suggestion
        [ActiveIssue("https://github.com/dotnet/runtime/issues/21918", TestPlatforms.AnyUnix)]
```

**Review by ViktorHofer — APPROVED** [review]
Thanks :)

**Comment by sdmaclea** [comment]
Turns out doing this with a regular expression is more difficult than imagined.  The conditions need to be inverted.  I think the risk exceeds the benefit....  I am going to close this...

**Pull request closed without merging**